SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -1,137 +1,332 @@
+                    w.WriteLine($"{ipaddress} {hostname}");
+                }
+            }
+            else if (FdOs.IsLinux())
+            {
+                DockerHelper.ExecuteBashCommand($"echo {ipaddress} {hostname} | sudo tee -a /etc/hosts");
             }
         }
-    }
 
-    [Binding]
-    public class Hooks
-    {
-        private readonly IObjectContainer ObjectContainer;
-        private IWebDriver WebDriver;
+        /// <summary>
+        /// Executes the bash command.
+        /// </summary>
+        /// <param name="command">The command.</param>
+        /// <returns></returns>
+        private static String ExecuteBashCommand(String command)
+        {
+            // according to: https://stackoverflow.com/a/15262019/637142
+            // thans to this we will pass everything as one command
+            command = command.Replace("\"", "\"\"");
+
+            var proc = new Process
+                       {
+                           StartInfo = new ProcessStartInfo
+                                       {
+                                           FileName = "/bin/bash",
+                                           Arguments = "-c \"" + command + "\"",
+                                           UseShellExecute = false,
+                                           RedirectStandardOutput = true,
+                                           CreateNoWindow = true
+                                       }
+                       };
+            Console.WriteLine(proc.StartInfo.Arguments);
+
+            proc.Start();
+            proc.WaitForExit();
+
+            return proc.StandardOutput.ReadToEnd();
+        }
 
-        public Hooks(IObjectContainer objectContainer)
+        /// <summary>
+        /// Setups the security service container.
+        /// </summary>
+        /// <param name="containerName">Name of the container.</param>
+        /// <param name="logger">The logger.</param>
+        /// <param name="imageName">Name of the image.</param>
+        /// <param name="networkService">The network service.</param>
+        /// <param name="hostFolder">The host folder.</param>
+        /// <param name="dockerPort">The docker port.</param>
+        /// <param name="dockerCredentials">The docker credentials.</param>
+        /// <param name="forceLatestImage">if set to <c>true</c> [force latest image].</param>
+        /// <param name="additionalEnvironmentVariables">The additional environment variables.</param>
+        /// <returns></returns>
+        private static IContainerService SetupSecurityServiceContainer(String containerName,
+                                                                       ILogger logger,
+                                                                       String imageName,
+                                                                       INetworkService networkService,
+                                                                       String hostFolder,
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'hostFolder'.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)